### ISSUE TITLE
Bulk unpublish PMID:36261456

### ISSUE THREAD

**Issue by Santhi1901** [EBISPOT/goci#917]
The studies need to be unpublished to update EFO. There are 150 studies in this PMID

**Comment by sajo-ebi** [comment]
@Santhi1901 the studies has been unpublished

**Comment by Santhi1901** [comment]
@sajo-ebi. I have now updated the EFOs for this PMID. Could you please re-publish the studies (except the study id:101064081)

**Comment by sajo-ebi** [comment]
@Santhi1901 Published kindly confirm , Also validate if studyId 101064081 is not published

**Comment by Santhi1901** [comment]
Yes, all the required studies are published, and study id 101064081 is not published. Thank you.
